jsk_aibo_robot/aibo_selenium_ros/node_scripts/main.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+
+import logging
+import sys
+import time
+
+import cv2
+import rospy
+from cv_bridge import CvBridge
+from sensor_msgs.msg import CompressedImage, Image
+
+from aibo_selenium_ros import AIBOBrowserInterface
+
+if __name__ == '__main__':
+
+  rospy.init_node('aibo_selenium_ros')
+
+  pub_raw = rospy.Publisher('~image', Image, queue_size=1)
+  pub_compressed = rospy.Publisher('~image/compressed', CompressedImage, queue_size=1)
+  cv_bridge = CvBridge()
+
+  logging.basicConfig(level=logging.INFO)
+  logger = logging.getLogger(__name__)
+
+  webdriver = rospy.get_param('~webdriver')
```

Comment:
you can use `/usr/bin/chromedriver` as a default value.

Reply:
Set.